src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+
+    public RightTriangle(double firstLeg,double secondLeg, Color color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        double result = (firstLeg * firstLeg) + (secondLeg * secondLeg);
+        return Math.sqrt(result);
```

Comment:
it's hypotenuse formula from Pifagors theorem. instead, you should calculate triangle area - so formula is invalid